### ISSUE TITLE
Colorize argument names

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#31876]
<p>Currently argument names and arguments are white. It becomes crowded and hard to follow especially when there are multiple arguments. Argument names should have their own color for clarity.</p><p><a href="https://developercommunity.visualstudio.com/storage/attachments/51416-vs-argument-names.jpg"> Image:51416-vs-argument-names.jpg</a></p>

_This issue has been moved from https://developercommunity.visualstudio.com/content/idea/402436/colorize-argument-names.html
VSTS ticketId: 743884_
_These are the original issue comments:_

Jane Wu [MSFT] on 12/9/2018, 08:27 PM (7 days ago): <p>Thank you for taking the time to provide your suggestion. We will do some preliminary checks to make sure we can proceed further.&nbsp;You will hear from us in about a week on our next steps.</p>

**Comment by Neme12** [comment]
I don't think this is quite a duplicate of that. This is a request for classification of any kind of symbol name (for example a field) in place of an argument. The part determining the classification would the the fact that it's inside an argument, not the type of the symbol.

**Comment by Neme12** [comment]
@sharwell

**Comment by sharwell** [comment]
Oh, thanks, I see!

**Comment by Neme12** [comment]
Although I feel like this could easily be addressed by giving methods a separate color from all other symbols.

**Comment by CyrusNajmabadi** [comment]
Yeah, i would prefer to just go with @JoeRobich's solution here.  Color the item based on what it is, not how it is used.

**Comment by sharwell** [comment]
Tentatively marking as **won't fix**, but will update following design review.

**Comment by sharwell** [comment]
Design review conclusion: named parameters at call sites are colorized as parameters starting in Visual Studio 2019. The color for parameters defaults to the same as locals, but users may customize it to something else. We are treating this issue as **Fixed**.
